### PR TITLE
Style cleanup for `modules/data.mining/inst/*.R` scripts 

### PR DESCRIPTION
**Pull request by bpbond**
These scripts are pretty rough; I cleaned up but didn’t spend a huge amount of time on them.

Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure all if blocks have braces. Tried to make sure formatR didn’t make things too jaggy. Substituted `seq_len`, `seq_along`, and `paste0`, and `file.path` as necessary.

Note `NACPspecPost.v2.R` has syntax errors (doesn’t even source). I have ignored it for now.

### CODE REVIEW

File: modules/data.mining/inst/NACPspectral.R [hunk 1]
```diff
@@ -1,268 +1,242 @@
...
+  
+  ## histogram approach (% of significant failure events)
+  
+  i
+  ## analysis of spectra - mixed model
+  
+  ## TODO: - define model set - define time-step - define sites - incorporation of uncertainty - gaps
+  ## or gap-filled - alternative visualizations - identification of whether model failures are
+  ## consistent in time - separation of pattern from accuracy - look at cross-spectra?  - absolute vs
+  ## normalised residuals
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspectral.R]: incorrectly flattened

File: modules/data.mining/inst/NACPspectral.R [hunk 2]
```diff
@@ -1,268 +1,242 @@
-## Code to analyse the spectral signal of model error
-## between models and fluxtowers
-##
-## Analysis part of the NACP site-level intercomparison
-##
-## Michael Dietze, University of Illinois
-##
+## Code to analyse the spectral signal of model error between models and fluxtowers Analysis part
+## of the NACP site-level intercomparison Michael Dietze, University of Illinois
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspectral.R]: flattened

File: modules/data.mining/inst/ModelResidualSpectra.R [hunk 3]
```diff
@@ -1,7 +1,4 @@
-## Script to analyse the spectral signal of model error
-##
-## Michael Dietze, Boston University 
-##
+## Script to analyse the spectral signal of model error Michael Dietze, Boston University
```
Comment (mdietze) [inline modules/data.mining/inst/ModelResidualSpectra.R]: flattened

File: modules/data.mining/inst/NACPspecPost.R [hunk 4]
```diff
@@ -1,13 +1,9 @@
-## Code to post-process (summarize and graph) the spectral
-## signal of model error between models and fluxtowers
-##
-## Analysis part of the NACP site-level intercomparison
-##
-## Michael Dietze, University of Illinois
-##
+## Code to post-process (summarize and graph) the spectral signal of model error between models and
+## fluxtowers Analysis part of the NACP site-level intercomparison Michael Dietze, University of
+## Illinois
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspecPost.R]: flattened

File: modules/data.mining/inst/NACPspectral.Keenan.R [hunk 5]
```diff
@@ -1,164 +1,160 @@
-## Code to analyse the spectral signal of model error
-## between models and fluxtowers
-##
-## Analysis part of the NACP site-level intercomparison
-##
-## Michael Dietze, University of Illinois
-##
+## Code to analyse the spectral signal of model error between models and fluxtowers Analysis part
+## of the NACP site-level intercomparison Michael Dietze, University of Illinois
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspectral.Keenan.R]: flattened

File: modules/data.mining/inst/NACPspectral.v2.R [hunk 6]
```diff
@@ -1,280 +1,265 @@
-## Code to analyse the spectral signal of model error
-## between models and fluxtowers
-##
-## Analysis part of the NACP site-level intercomparison
-##
-## Michael Dietze, University of Illinois
-##
+## Code to analyse the spectral signal of model error between models and fluxtowers Analysis part
+## of the NACP site-level intercomparison Michael Dietze, University of Illinois
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspectral.v2.R]: flattened

File: modules/data.mining/inst/NACPspectral.v2.R [hunk 7]
```diff
@@ -1,280 +1,265 @@
...
+    
+    ## histogram approach (% of significant failure events)
+    
+    i
+    ## analysis of spectra - mixed model
+    
+    ## TODO: - define model set - define time-step - define sites - incorporation of uncertainty - gaps
+    ## or gap-filled - alternative visualizations - identification of whether model failures are
+    ## consistent in time - separation of pattern from accuracy - look at cross-spectra?  - absolute vs
+    ## normalised residuals
```
Comment (mdietze) [inline modules/data.mining/inst/NACPspectral.v2.R]: flattened

File: modules/data.mining/inst/NullSpectra.R [hunk 8]
```diff
@@ -1,166 +1,148 @@
...
-###
-### Michael Dietze
-###
-### inputs: annual NEE matrices of [time x ensemble member] 
-### outputs: mean and quantile spectra
+### generate wavelet spectra of eddy flux data that incorporates uncertainty This code is
+### computationally intensive so it is set up to be submitted to a cluster queue assumes that it is
+### passed an ID for which site is being analyzed and a number range of Monte Carlo iterations to
+### process Michael Dietze inputs: annual NEE matrices of [time x ensemble member] outputs: mean and
+### quantile spectra
```
Comment (mdietze) [inline modules/data.mining/inst/NullSpectra.R]: flattened

File: modules/data.mining/inst/NullSpectra.v2.R [hunk 9]
```diff
@@ -1,218 +1,210 @@
 
-### generate wavelet spectra of the flux data that incorporates
-### random and gap-filling uncertainty based on the Barr et al
-### NACP data product
+### generate wavelet spectra of the flux data that incorporates random and gap-filling uncertainty
+### based on the Barr et al NACP data product
 
-### inputs: annual NEE matrices of [time x ensemble member] 
-### outputs: mean and quantile spectra
+### inputs: annual NEE matrices of [time x ensemble member] outputs: mean and quantile spectra
```
Comment (mdietze) [inline modules/data.mining/inst/NullSpectra.v2.R]: flattened

File: modules/data.mining/inst/NullSpectraPost.R [hunk 10]
```diff
@@ -1,88 +1,82 @@
 
-### generate wavelet spectra of the flux data that incorporates
-### random and gap-filling uncertainty based on the Barr et al
-### NACP data product
+### generate wavelet spectra of the flux data that incorporates random and gap-filling uncertainty
+### based on the Barr et al NACP data product
 
-### inputs: annual NEE matrices of [time x ensemble member] 
-### outputs: mean and quantile spectra
+### inputs: annual NEE matrices of [time x ensemble member] outputs: mean and quantile spectra
```
Comment (mdietze) [inline modules/data.mining/inst/NullSpectraPost.R]: flattened

File: modules/data.mining/inst/NullSpectraPost.v2.R [hunk 11]
```diff
@@ -1,90 +1,81 @@
 
-### generate wavelet spectra of the flux data that incorporates
-### random and gap-filling uncertainty based on the Barr et al
-### NACP data product
+### generate wavelet spectra of the flux data that incorporates random and gap-filling uncertainty
+### based on the Barr et al NACP data product
 
-### inputs: annual NEE matrices of [time x ensemble member] 
-### outputs: mean and quantile spectra
+### inputs: annual NEE matrices of [time x ensemble member] outputs: mean and quantile spectra
```
Comment (mdietze) [inline modules/data.mining/inst/NullSpectraPost.v2.R]: flattened

**Comment by bpbond** [comment]
Argh. Sorry about missing all these! I'm not sure why dived into `inst/`...probably not useful. I'll update the PR, though feel free to just close this if you don't want to bother.

**Pull request merged**